gapis/perfetto/android/trace.go

```diff
@@ -208,3 +234,76 @@ func (p *Process) Capture(ctx context.Context, start task.Signal, stop task.Sign
 	}
 	return io.Copy(w, fh)
 }
+
+func (p *Process) captureWithClientApi(ctx context.Context, start task.Signal, stop task.Signal, ready task.Task, w io.Writer, written *int64) (int64, error) {
+	defer p.perfettoClient.Close(ctx)
+
+	p.config.DeferredStart = proto.Bool(true)
+	var buf bytes.Buffer
+	ts, err := p.perfettoClient.Trace(ctx, p.config, &buf)
+	if err != nil {
+		return 0, log.Err(ctx, err, "Failed to call EnableTrace")
```

Comment:
The message shouldn't mention `EnableTrace`, that is a client internal and is likely already part of the error. Simply "Failed to setup Perfetto trace" or something like that, is better.